fastlane/lanes/screenshots.rb

```diff
@@ -0,0 +1,230 @@
+  ['ar', 'ar-SA'],
+  ['en-gb', 'en-US'],
+  ['en-gb', 'en-GB'],
+  ['en-ca', 'en-CA'],
+  ['en-au', 'en-AU'],
+  ['da', 'da'],
+  ['de', 'de-DE'],
+  ['es', 'es-ES'],
+  ['fr', 'fr-FR'],
+  ['he', 'he'],
+  ['id', 'id'],
+  ['it', 'it'],
+  ['ja', 'ja'],
+  ['ko', 'ko'],
+  ['nl', 'nl-NL'],
+  ['nb', 'no'],
+  ['pt-br', 'pt-BR'],
+  ['pt', 'pt-PT'],
+  ['ru', 'ru'],
+  ['sv', 'sv'],
+  ['th', 'th'],
+  ['tr', 'tr'],
+  ['zh-cn', 'zh-Hans'],
+  ['zh-tw', 'zh-Hant']
+].freeze
+# rubocop:enable Style/WordArray
+
+#################################################
+# Lanes
+#################################################
+
+# Lanes related to Generating Screenshots
+#
+platform :ios do
+  # Generates the localized raw screenshots (for WordPress by default), using the dedicated UI test scheme
+  #
+  # Generates both light and dark mode, for each locale, and a fixed set of device sizes (2 iPhones, 2 iPads)
+  #
+  # @option [String] scheme (default: "WordPressScreenshotGeneration") The name of the scheme to use to run the screenshots
+  # @option [String] output_directory (default: `${PWD}/screenhots`) The directory to generate the screenshots to
+  # @option [Array<String>] language (default: the list of Mag16 locales) The subset of locales to generate the screenshots for
+  #
+  desc 'Generate localised screenshots'
+  lane :screenshots do |options|
+    sh('bundle exec pod install')
+    FileUtils.rm_rf(DERIVED_DATA_PATH)
+
+    scheme = options[:scheme] || 'WordPressScreenshotGeneration'
+
+    output_directory = options[:output_directory] || File.join(Dir.pwd, '/screenshots')
+
+    scan(
+      workspace: WORKSPACE_PATH,
+      scheme: scheme,
+      build_for_testing: true,
+      derived_data_path: DERIVED_DATA_PATH
+    )
+
+    languages = %w[ar-SA da de-DE en-AU en-CA en-GB es-ES fr-FR he id it ja ko no nl-NL pt-BR pt-PT ru sv th tr zh-Hans zh-Hant en-US]
+
+    # Allow creating screenshots for just one languages
+    unless options[:language].nil?
+      languages.keep_if do |language|
+        language.casecmp(options[:language]).zero?
+      end
+    end
+
+    puts languages
+
+    [true, false].each do |dark_mode_enabled|
+      capture_ios_screenshots(
+        workspace: WORKSPACE_PATH,
+        scheme: scheme,
+        test_without_building: true,
+        derived_data_path: DERIVED_DATA_PATH,
+        output_directory: output_directory,
+        languages: languages,
+        dark_mode: dark_mode_enabled,
+        override_status_bar: true,
+
+        reinstall_app: true,
+        erase_simulator: true,
+        localize_simulator: true,
+        concurrent_simulators: true,
+
+        devices: [
+          'iPhone Xs Max',
+          'iPhone 8 Plus',
+          'iPad Pro (12.9-inch) (2nd generation)',
+          'iPad Pro (12.9-inch) (3rd generation)'
+        ]
+      )
+    end
+  end
+
+  # Generates the localized raw screenshots for Jetpack, using the dedicated `"JetpackScreenshotGeneration"` UI test scheme
```

Comment:
Nitpick: The `""` seem unnecessary when the scheme name is already within backticks.

```suggestion
  # Generates the localized raw screenshots for Jetpack, using the dedicated `JetpackScreenshotGeneration` UI test scheme
```